cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaClusterRebalanceAssemblyOperator.java

```diff
@@ -269,28 +279,28 @@ public void onClose(KubernetesClientException e) {
 
         return computeNextStatus(reconciliation, host, apiClient, clusterRebalance, currentState, rebalanceAnnotation, rebalanceOptionsBuilder)
                 .compose(desiredStatus -> {
-                    // due to a long rebalacing operation that takes the lock for the entire period, more events related to resource modification could be
-                    // queued with a stale resource (updated by the rebalacing holding the lock), so we need to get the current fresh resource
+                    // due to a long rebalancing operation that takes the lock for the entire period, more events related to resource modification could be
+                    // queued with a stale resource (updated by the rebalancing holding the lock), so we need to get the current fresh resource
                     return clusterRebalanceOperator.getAsync(reconciliation.namespace(), reconciliation.name())
-                            .compose(freshClusterRebalance -> {
-                                return updateStatus(freshClusterRebalance, desiredStatus, clusterRebalanceOperator, null)
-                                        .compose(c -> {
-                                            log.info("{}: State updated to [{}] with annotation {}={} ",
-                                                    reconciliation, c.getStatus().getConditions().get(0).getType(), ANNO_STRIMZI_IO_REBALANCE,
+                            .compose(freshClusterRebalance -> updateStatus(freshClusterRebalance, desiredStatus, clusterRebalanceOperator, null)
```

Comment:
just because github doesn't help, the only change here is using an expression lambda instead of a statement lambda so removing the "return" right?

Reply:
Yes that is correct